src/nni_manager/training_service/remote_machine/test/shellExecutor.test.ts

```diff
@@ -41,14 +44,16 @@ describe('ShellExecutor test', () => {
         rmMeta = JSON.parse(fs.readFileSync('../../.vscode/rminfo.json', 'utf8'));
         console.log(rmMeta);
     } catch (err) {
-        console.log(`Please configure rminfo.json to enable remote machine test.${err}`);
+        console.log(`Please configure rminfo.json to enable remote machine test. ${err}`);
```

Comment:
remove console.log() in TS code, use `this.log()` instead.

Reply:
tried, it doesn't work in test code.